src/new-client/src/App/Trades/TradesCore.tsx

```diff
@@ -7,28 +7,51 @@ import { TradesHeader } from "./TradesHeader"
 import { tableTrades$ } from "./TradesState"
 import { createSuspenseOnStale } from "@/utils/createSuspenseOnStale"
 import { isBlotterDataStale$ } from "@/services/trades"
+import { useCallback, useState } from "react"
 
 const TradesStyle = styled.div`
   height: 100%;
   width: 100%;
   color: ${({ theme }) => theme.core.textColor};
   font-size: 0.8125rem;
+  position: relative;
 `
 
 const SuspenseOnStaleData = createSuspenseOnStale(isBlotterDataStale$)
+const Trades: React.FC = () => {
+  const [height, setHeight] = useState<number>(0)
+  const [width, setWidth] = useState<number>(0)
+  const ref = useCallback((node) => {
+    let resizeObserver = new ResizeObserver(() => {})
```

Comment:
Rather than instantiating a `ResizeObserver` twice, you can do it once here and observe/disconnect in the if/else.

Reply:
fixed